### PR TITLE
new commit

### PR DESCRIPTION
**Pull request by tchasinga**
Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
Create a new GitHub repo.
Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
Add a .gitignore file.
Add node_modules to gitignore file.
Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Add 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,271 @@
...
+ 
+ - Example :
+   - npm run start
+   - git status
+   - git clone 
+   - npm i (+ the name of specific element )
+
+>### Setup
+
+Clone this repository to your desired folder:
```
Comment (Olamarx) [inline README.md:126]: - [OPTIONAL] Which repository should they clone? I will advise you to add the url.

File: README.md [hunk 2]
```diff
@@ -1 +1,271 @@
...
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```
Comment (Olamarx) [inline README.md]: - [ ] I will advise you to remove this Microverse dummy data and add yours if you have one, if you don't, just remove it and add the name of the project.

File: README.md [hunk 3]
```diff
@@ -1 +1,271 @@
...
+
+- GitHub: [@tchasinga](https://github.com/tchasinga)
+- Twitter: [@tchasinga](https://twitter.com/tchasinga)
+- LinkedIn: [tchasinga](https://linkedin.com/in/tchasinga)
+
+👤 **Jonatha**
+
+- GitHub: [@Jonatha](https://github.com/Jonatha)
+- Twitter: [@Jonatha](https://twitter.com/Jonatha)
+- LinkedIn: [Jonatha](https://linkedin.com/in/Jonatha)
```
Comment (Olamarx) [inline README.md]: - [ ] I think this is a solo project, you don't have a partner so you should not have two authors.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @tchasinga, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] As required, I will advise you to create a test.md file and add it to gitIgnore file so that the file won't show in the repo.

image

- [ ]  I will implore you to always name your feature branch according to what you are doing in the branch such as 'created_sidebar' or 'added_comments_section', in this case, it can be 'microverse_hello_world' or anything related. To achieve this, kindly checkout to a new branch and use the correct name and mention this old PR in the new PR.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @tchasinga 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Right structure used ✔️
- right `.gitignore` files👍 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**